test/functional/command/test_install_deploy.py

```diff
@@ -511,6 +511,39 @@ def package_info(self):
         expected = sorted(["pkga.so", "pkgb.so", "pkga.dll"])
         assert sorted(os.listdir(os.path.join(c.current_folder, "myruntime"))) == expected
 
+@pytest.mark.parametrize("symlink, expected",
+                         [(True, ["libfoo.so.0.1.0", "libfoo.so.0", "libfoo.so"]),
+                          (False, ["libfoo.so.0.1.0",])])
+def test_runtime_deploy_symlinks(symlink, expected):
```

Comment:
Shouldnt this test be skipped for Windows?

Reply:
It could be skipped for Windows, but keeping it we make sure we are not breaking in Windows as a regression test.

Reply:
Isee, this test will fail in Windows machines without symlinks enabled:
```
>>> os.symlink("potato", "mypotato")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OSError: [WinError 1314] A required privilege is not held by the client
```
It passes because in our CI maybe we enabled it, it is not the default.
But ok, if it works in Windows if symlinks enabled, good then, I'll try to remind it when running locally.

Reply:
A comment will be nice

Reply:
Should we protect users by checking if symlink is enabled in Windows and avoid copying symlinks. Or, raise a ConanException when symlinks=True and Windows symlinks is not enabled. At least, it would not break with a OSError for users.